src/pages/Products/Products.js

```diff
@@ -1,7 +1,50 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
+import ProductsFilter from "./Filter/ProductsFilter";
+import ProductCard from "./ProductCard/ProductCard";
+import "./Products.scss";
 
 const Products = () => {
-  return <div>상품 리스트</div>;
+  const [products, setProducts] = useState([]);
+
+  useEffect(() => {
+    fetch("http://localhost:3000/data/Products/mockData.json", {
+      method: "GET",
+    })
+      .then(response => response.json())
+      .then(result => {
+        setProducts(result);
+      });
+  }, []);
```

Comment:
```suggestion
  useEffect(() => {
    fetch("/data/Products/mockData.json")
      .then(response => response.json())
      .then(result => {
        setProducts(result);
      });
  }, []);
```
http://localhost:3000 은 생략 가능합니다. 
또한 fetch함수의 get method는 method의 default 값입니다. 따라서 생략이 가능합니다.